gym_sts/envs/base.py

```diff
@@ -183,6 +185,9 @@ def _end_game(self) -> None:
         # If still alive
         self.communicator.resign()
 
+        obs = self.observe()
```

Comment:
You can just combine these, `obs = self.communicator.resign()`.

Reply:
done